lib/config.yml

```diff
@@ -127,6 +127,12 @@ RSpec/VerifiedDoubles:
 Style/ClassCheck:
   Enabled: false
 
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
```

Comment:
Feels like we should leave this one. im not sure what will need to change but it will be interesting. 

From cop site:
```
This cop is designed to help upgrade to Ruby 3.0. It will add the comment # frozen_string_literal: true to the top of files to enable frozen string literals. Frozen string literals will be default in Ruby 3.0. The comment will be added below a shebang and encoding comment. The frozen string literal comment is only valid in Ruby 2.3+
```